doc/cookbook/source/templates/page.html

```diff
@@ -23,6 +23,17 @@
 });
 </script>
 
+<script type="text/x-mathjax-config">
+  MathJax.Hub.Config({
+    TeX: {
+      Macros: {
```

Comment:
i am not sure this was working before. What is the rationale here?

Reply:
Currently, if you open [shogun.ml/examples][shogun-example], you find the JS error below:
```
Uncaught ReferenceError: MathJax is not defined
    at mathconf.js:1
```

This error happens because `mathconf.js` is loaded before the core MathJax script, which contains the `MathJax` variable. Hence,  macros defined in `mathconf.js`, such as `\argmax`,  are not defined.  

In order to solve this issue, we need to make sure that `mathconf.js` is executed after the core MathJax script is loaded. Including `mathconf.js` with `type="text/x-mathjax-config"` is the solution.  I think the import below might also work and look nicer:

```
<script type="text/x-mathjax-config" src="_static/mathconf.js"></script>
```



[shogun-example]: http://shogun.ml/examples/latest/index.html